package.json

```diff
@@ -31,11 +31,14 @@
   },
   "dependencies": {
     "anchor-js": "^4.2.2",
+    "assert": "^2.0.0",
```

Comment:
I'm not a fan of too many dev deps, can you try can you try https://www.npmjs.com/package/node-polyfill-webpack-plugin?

Reply:
There will be more dependencies if you use that plugin: https://github.com/Richienb/node-polyfill-webpack-plugin/blob/master/package.json#L26-L49 , because it provides all Node.js built-in modules which most of them we don't use.

Reply:
I think it's not a big deal - unused modules will not be bundled.

Reply:
This isn't related to your concern: you're care about "many dev dependencies".

Reply:
node-polyfill-webpack-plugin is just 1 dependency.

Reply:
Add those modules (`assert`, `path-browserify` and `process`) can be listed clearly and for better management, while installing `node-polyfill-webpack-plugin` will bring a bunch of dependencies, even webpack doesn't bundle them.

Reply:
I agreed it is reasonable for web apps, while for node.js libs, it's not.

it will add a large maintenance burden - eslint is likely to use more built-in modules, we have to update the repo too. this is a kind of ...

Reply:
I think it would be easier for maintenance to have one dependency that handles everything, than to track which built-in Node.js modules are required. I guess we also can't know whether ESLint's dependencies that will be bundled are using some additional built-in modules.

Reply:
Waiting for: https://github.com/Richienb/node-polyfill-webpack-plugin/pull/3